modules/cert-manager-acme-dns01-aws.adoc

```diff
@@ -0,0 +1,170 @@
+
+.. Create the `Namespace` object by running the following command:
++
+[source,terminal]
+----
+$ oc create -f namespace.yaml
+----
+
+. Create a secret to store your AWS credentials in by running the following command:
++
+[source,terminal]
+----
+$ oc create secret generic aws-secret --from-literal=awsSecretAccessKey=<aws_secret_access_key> \ <1>
+    -n my-issuer-namespace
+----
+<1> Replace `<aws_secret_access_key>` with your AWS secret access key.
+
+. Create an issuer.
+
+.. Create a YAML file that defines the `Issuer` object:
++
+.Example `issuer.yaml` file
+[source,yaml]
+----
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: letsencrypt-staging                                        <1>
+  namespace: my-issuer-namespace                                   <2>
+spec:
+  acme:
+    server: https://acme-staging-v02.api.letsencrypt.org/directory <3>
+    email: "<email_address>"                                       <4>
+    privateKeySecretRef:
+      name: <secret_private_key>                                   <5>
+    solvers:
+    - dns01:
+        route53:
+          accessKeyID: <aws_key_id>                                <6>
+          hostedZoneID: <hosted_zone_id>                           <7>
+          region: us-east-1
+          secretAccessKeySecretRef:
+            name: "aws-secret"                                     <8>
+            key: "awsSecretAccessKey"                              <9>
+----
+<1> Provide a name for the issuer.
+<2> Specify the namespace that you created for the issuer.
+<3> Specify the URL to access the ACME server's `directory` endpoint. This example uses the _Let's Encrypt_ staging environment.
+<4> Replace `<email_address>` with your email address.
+<5> Replace `<secret_private_key>` with the name of the secret to store the ACME account private key in.
+<6> Replace `<aws_key_id>` with your AWS key ID.
+<7> Replace `<hosted_zone_id>` with your hosted zone ID.
+<8> Specify the name of the secret you created.
+<9> Specify the key in the secret you created that stores your AWS secret access key.
+
+.. Create the `Issuer` object by running the following command:
++
+[source,terminal]
+----
+$ oc create -f issuer.yaml
+----
+
+. Create a certificate.
+
+.. Create a YAML file that defines the `Certificate` object:
++
+.Example `certificate.yaml` file
+[source,yaml]
+----
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: my-tls-cert              <1>
+  namespace: my-issuer-namespace <2>
+spec:
+  isCA: false
+  commonName: '<common_name>'    <3>
+  secretName: my-tls-cert        <4>
+  dnsNames:
+  - '<domain_name>'              <5>
+  issuerRef:
+    name: letsencrypt-staging    <6>
+    kind: Issuer
+----
+<1> Provide a name for the certificate.
+<2> Specify the namespace that you created for the issuer.
+<3> Replace `<common_name>` with your common name (CN).
+<4> The name of the secret to create that will contain the certificate.
```

Comment:
Should <4> start with "Replace" or "Specify" as the other notes do? For example, "Specify the name of the secret to create that will contain the certificate."

Reply:
Updating!